### PR TITLE
Show command to approve versions in KIO

### PR DESCRIPTION
**Pull request by rafaelcaricio**
It is useful to show what are the next steps to allow application to be completely deployed.
/cc @lotharschulz

### CODE REVIEW

File: lizzy_client/cli.py [hunk 1]
```diff
@@ -133,6 +134,10 @@ def create(definition: str, image_version: str, keep_stacks: int,
             fatal_error('Deployment failed: {}'.format(last_state))
 
     info('Deployment Successful')
+    if app_version:
+        info('You can prove this new version using the command:\n\n\t'
```
Comment (lotharschulz) [inline lizzy_client/cli.py]: s/prove/approve/

**Comment by lotharschulz** [comment]
+1

File: lizzy_client/cli.py (hunk 1 above)
Comment (rafaelcaricio) [inline lizzy_client/cli.py]: typos...

**Comment by jmcs** [comment]
:+1:
Can you also change the version to 1.8? Then I can upload this to pypy when we release lizzy 1.8.

**Comment by rafaelcaricio** [comment]
@jmcs Version updated.

**Pull request merged**